### DISCUSSION TITLE
Backup retentions monthly / yearly policy?

### DISCUSSION THREAD

**Discussion by jehon** [tiredofit/docker-db-backup#375] [Ideas]
While your system is really great, I wonder if we could add a backup retention policy, looking like:
- keep daily backups for 120 days (ok, DEFAULT_CLEANUP_TIME can do this)
- keep 24 monthly backups
- keep 10 yearly backups

Good idea or not?

**Comment by tiredofit** [comment]
Possible to implement but it would be custom development to do so. I let my upstream backup software (I am using Restic) to handle the Weekly Monthly Yearly retention and rotations. I find its easier to follow along and leave the db backup containers to handle the dailies (with a few extra lying around in the event of failed upstream backups, or quick recovery) in my infrastructure.
